### PR TITLE
Update 01-introduction.md - "Linestyles and markers" section

### PR DESCRIPTION
**Pull request by amanmdesai**
This commit solves issue #63 Add example to the "Linestyles and markers" section

### CODE REVIEW

File: _episodes/01-introduction.md [hunk 1]
```diff
@@ -217,6 +217,21 @@ Here are a few example for combinations of these.
...
 ```
+# Exercise:
+
+### Exercise: Using the table above, plot functions using different linestyles/markers on the same canvas:
+> ## Solution
+> ```python
+> x1 = np.linspace(1, 10)
+> y1, y2, y3 = np.log(x1), np.cos(x1), np.sin(x1)
+> fig, ax = plt.subplots()  # Create a figure containing a single axes.
+> ax.plot(x1, y1, linestyle="--", label="Dashed line")  # Create
```
Comment (klieret) [inline _episodes/01-introduction.md]: Superfluous `# Create`

File: _episodes/01-introduction.md [hunk 2]
```diff
@@ -217,6 +217,21 @@ Here are a few example for combinations of these.
...
+> ax.plot(x1, y2, linestyle=":", label="Dotted line style")
+> ax.plot(x1, y3, marker="o", label="Circle marker")
+> ax.legend()
+> plt.show()
```
Comment (klieret) [inline _episodes/01-introduction.md]: This should be `fit.show()` with the OO API

**Review by klieret — CHANGES_REQUESTED** [review]

**Comment by klieret** [comment]
Thanks a lot for opening this @amanmdesai . 

(I also just realized that the other examples in the first lesson also still use `plt.XYZ` and aren't formatted properly, and opened #83 for this)

**Review by klieret — APPROVED** [review]

File: _episodes/01-introduction.md (hunk 2 above)
Comment (amanmdesai) [reply _episodes/01-introduction.md]: Hi Thanks for your comments. I have implemented both the comments.

**Comment by klieret** [comment]
@allcontributors-bot please add @amanmdesai for content

**Comment by klieret** [comment]
@all-contributors please add @amanmdesai for content

**Pull request merged**